### DISCUSSION TITLE
Best way to parse captions in `sendMediaGroup`?

### DISCUSSION THREAD

**Discussion by mkhoussid** [telegraf/telegraf#1791] [Q&A] [answered]
Suppose the following,

```
const mediaGroup = [
  {
    media: 'file_id1',
    type: 'photo',
    caption: '**My caption**'
  },
  {
    media: 'file_id2',
    type: 'photo',
  },
  {
    media: 'file_id3',
    type: 'photo',
  }
]

const [{ message_id: sentMessageId }] = await bot.telegram.sendMediaGroup(123456, mediaGroup, {});
```

The caption is sent, but it isn't parsed as bold text. How can I pass on `parse_mode: 'MarkdownV2'` with this approach? `sendMediaGroup` doesn't accept this parameter.

For now, I am storing `sentMessageId` in an array, and then making use of `editMessageCaption` with `parse_mode: 'MarkdownV2'` to manually style the text, but this isn't as clean as I'd like it to be.

Any thoughts?

**Comment by MKRhere** [comment, marked as answer]
`parse_mode` is accepted in each media object.

```TS
[
	{
		media: "file_id1",
		type: "photo",
		caption: "**My caption\\!**",
		parse_mode: "MarkdownV2",
	},
	...
]
```
